1.9/administration/metrics/index.md

```diff
@@ -3,4 +3,142 @@ post_title: Metrics
 menu_order: 3.5
 ---
 
+The [metrics component](https://github.com/dcos/dcos-metrics) provides operational insight to your DC/OS cluster, providing discrete metrics about your applications and deployments. This can include charts, dashboards, and alerts based on cluster, node, container, and application-level statistics. The metrics component is natively integrated with DC/OS version 1.9 and later. No additional setup is required.  
```

Comment:
2nd to last sentence: The metrics component is natively integrated with DC/OS version 1.9 and later and is available per-host from the `/system/v1/metrics/v0` HTTP API endpoint.